package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "alanlauer.com",
+  "version": "1.0.0",
+  "private": true,
+  "description": "alanlauer.com",
+  "author": "Brometheus",
+  "keywords": [
+    "gatsby"
+  ],
+  "scripts": {
+    "develop": "gatsby develop",
+    "start": "gatsby develop",
+    "build": "gatsby build",
+    "serve": "gatsby serve",
+    "clean": "gatsby clean"
+  },
+  "dependencies": {
+    "@mdx-js/react": "^2.3.0",
+    "gatsby": "^5.6.1",
+    "gatsby-plugin-mdx": "^5.7.0",
+    "gatsby-source-filesystem": "^5.7.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
+  }
```

Comment:
this is small request but a good habit to form. 

Can you specific the minimum node version using the 'engines' property in this package.json? 

you can see how to do it here. https://docs.npmjs.com/cli/v9/configuring-npm/package-json#engines 

and i believe the min node version we can use is  '>=18.0.0' as gatsby latest requires it. I was on node 16 which is why i picked this up.